resources/views/posts/index.blade.php

```diff
@@ -0,0 +1,81 @@
+@extends('posts.layout')
+
+@section('content')
+    <div class="container">
+        <div class="row">
+        <a class="btn btn-primary" href="{{ route('posts.create') }}">{{ __('messages.add_new_post') }}</a>
+            <table class="table">
+                <thead>
+                <tr>
+                    <th scope="col">#</th>
+                    <th scope="col">{{ __('messages.title') }}</th>
+                    <th scope="col">{{ __('messages.content') }}</th>
+                    <th scope="col">{{ __('messages.description') }}</th>
+                    <th scope="col">{{ __('messages.action') }}</th>
+                </tr>
+                </thead>
+                <tbody>
+                @foreach ($listPost as $post)
+                    <tr>
+                        <th scope="row">{{ $post->id }}</th>
+                        <td>{{ $post->title }}</td>
+                        <td>{{ $post->content }}</td>
+                        <td>{{ $post->description }}</td>
+                        <td>
+                            @if ( is_null($post->postTranslations()->translation()->first()) )
```

Comment:
Chỗ này postTranslations()->translation()->first() sẽ sinh ra câu query, trong khi đang ở trong vòng for vì vậy tránh query trong vòng for bạn nhé

Reply:
oki chuẩn rồi